src/syslog_message.ml

```diff
@@ -271,47 +273,45 @@ let parse_timestamp_rfc3164 s year =
     | _, _, _, _, None -> None
     | Some month, Some day, Some hour, Some min, Some sec ->
       match Ptime.of_date_time ((year, month, day), ((hour, min, sec), 0)) with
-      | Some ts ->
-          let data = String.with_range ~first:tslen ~len:(l - tslen) s in
-            Some (ts, data)
       | None -> None
+      | Some ts -> Some (ts, with_range ~first:tslen ~len:(l - tslen) s)
 
 let parse_timestamp s (ctx : ctx) =
   let ((year, _, _), _) = Ptime.to_date_time ctx.timestamp in
   match parse_timestamp_rfc3164 s year with
   | Some (timestamp, data) -> Some (timestamp, data, ctx)
   | None ->
-      let ctx = ctx_set_hostname ctx in
-        Some (ctx.timestamp, s, ctx)
+    let ctx = ctx_set_hostname ctx in
+    Some (ctx.timestamp, s, ctx)
 
 let parse_hostname s ctx =
-  if ctx.set_hostname
-  then
+  if ctx.set_hostname then
     Some (ctx.hostname, s)
   else
     match String.length s with
     | l when l > 1 ->
-        (match String.find (fun x -> x = ' ') s with
+      begin match String.find (fun x -> x = ' ') s with
```

Comment:
Not a big fan of this particular change as it makes it tougher for editors to match `begin`/`end` together than `(`/`)`.